dev/script/generate-config/service-config.js

```diff
@@ -142,6 +144,7 @@ function getThrallConfig(config) {
         |aws.region="${config.AWS_DEFAULT_REGION}"
         |s3.image.bucket="${config.coreStackProps.ImageBucket}"
         |s3.thumb.bucket="${config.coreStackProps.ThumbBucket}"
+        |s3.quarantine.bucket="${config.coreStackProps.QuarantineBucket}"
```

Comment:
hopefully this can be removed if `sendToQuarantine` is extracted into a `ImageQuarantineOperations` as suggested above.